common.js

```diff
@@ -254,5 +257,9 @@ module.exports = {
     fs.stat(filename, function (err) {
       cb(err, err ? null : filename)
     })
+  },
+
+  sanitizeExeFilename: function (name) {
```

Comment:
`exe` sounds awfully Windows-centric. `sanitizeExecutableFilename`?

Could you also write some tests for this function? (That don't involve running `packager()`.)